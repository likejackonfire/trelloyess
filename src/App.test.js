import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

it("renders without crashing", () => {
    // Create a DOM element to render the component 
    const div = document.createElement("div");
  
    // Render the component
    // If something is wrong it will fail here
    ReactDOM.render(<App />, div);
    // Clean up
    ReactDOM.unmountComponentAtNode(div);
  });

 //screenshot test
 it('renders the UI as expected', () => {
    const tree = renderer
      .create( <App name="app" /> )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });