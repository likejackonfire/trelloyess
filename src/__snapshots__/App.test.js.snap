// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<main
  className="App"
>
  <header
    className="App-header"
  >
    <h1>
      Trelloyes!
    </h1>
  </header>
  <div
    className="App-list"
  >
    <section
      className="List"
    >
      <header
        className="List-header"
      >
        <h2>
          First list
        </h2>
      </header>
      <div
        className="List-cards"
      >
        <div
          className="Card"
        >
          <p>
            a
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            First card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            b
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Second card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            e
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Fifth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            f
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Sixth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            g
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Seventh card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            j
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Tenth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            l
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Twelfth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            m
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Thirteenth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <button
          className="List-add-button"
          onClick={[Function]}
          type="button"
        >
          + Add Random Card
        </button>
      </div>
    </section>
    <section
      className="List"
    >
      <header
        className="List-header"
      >
        <h2>
          Second list
        </h2>
      </header>
      <div
        className="List-cards"
      >
        <div
          className="Card"
        >
          <p>
            b
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Second card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            c
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Third card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            d
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Fourth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            f
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Sixth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            h
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Eighth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            i
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Ninth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            k
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Eleventh card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <button
          className="List-add-button"
          onClick={[Function]}
          type="button"
        >
          + Add Random Card
        </button>
      </div>
    </section>
    <section
      className="List"
    >
      <header
        className="List-header"
      >
        <h2>
          Third list
        </h2>
      </header>
      <div
        className="List-cards"
      >
        <div
          className="Card"
        >
          <p>
            a
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            First card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            b
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Second card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            c
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Third card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            d
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Fourth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            e
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Fifth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            f
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Sixth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            g
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Seventh card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            h
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Eighth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            i
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Ninth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            j
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Tenth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            k
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Eleventh card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            l
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Twelfth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            m
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Thirteenth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <button
          className="List-add-button"
          onClick={[Function]}
          type="button"
        >
          + Add Random Card
        </button>
      </div>
    </section>
    <section
      className="List"
    >
      <header
        className="List-header"
      >
        <h2>
          Fourth list
        </h2>
      </header>
      <div
        className="List-cards"
      >
        <div
          className="Card"
        >
          <p>
            l
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Twelfth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <div
          className="Card"
        >
          <p>
            m
          </p>
          <button
            onClick={[Function]}
            type="button"
          >
            Delete
          </button>
          <h3>
            Thirteenth card
          </h3>
          <p>
            lorem ipsum
          </p>
        </div>
        <button
          className="List-add-button"
          onClick={[Function]}
          type="button"
        >
          + Add Random Card
        </button>
      </div>
    </section>
  </div>
</main>
`;
